blog/exporters.md

```diff
@@ -74,48 +78,102 @@ The `secadvisor.yml` is used as a parameter when running secadvisor from the com
       # iam_endpoint: https://iam.cloud.ibm.com/identity/token
       object_prefix: logs
+
       # -- bulk store params --
       max_flows_per_object: 6000
       max_seconds_per_object: 60
       max_seconds_per_stream: 86400
       max_flow_array_size: 100000
 ```
 
-### Description of fields in `secadvisor.yml` file
+### Description of fields in `exporter.yml` file
+
+#### Synopsis
+
+| Field Name            | Type           | Description |
+|-----------------------|----------------|-------------|
+| host_id               | string         | "" or unique identifier |
+| analyzers             | URL            | address of Skydive analyzer for queries |
+| pipeline              | list of params | pipeline specification |
+| analyzer              | list of params | analyzer credentials for websocket connection |
+| subscriber_url        | URL            | websocket address to obtain flows from Skydive |
+| subscriber_username   | string         | websocket creedentials |
+| subscriber_password   | string         | websocket creedentials |
+| cluster_net_masks     | list of cidr   | list of net-addr/net_mask to be considered internal addresses |
+| filter                | list of params | filter options; currently the only parameter is `excluded_tags` |
+| excluded_tags         | list of tags   | specifies which flows to remove from report; possible values: `other`, `internal`, `ingress`, `egress` |
+| transform             | list of params | specifies how to transform the data; currently defined transforms are: `sa`, `awsflowlogs`, `vpclogs` |
+| sa                    | list of params | use Security Advisor transform; currently defined parameters are: `exclude_started_flows` |
+| exclude_started_flows | bool           | set to `true` if you want to exclude previously started flows from the report | 
+| encode                | list of params | encode options |
+| encode.type           | list of params | may be json or csv |
+| encode.json           | list of params | json options |
+| encode.json.pretty    | bool           | specifies whether json should be pretty-printed |
+| compress              | list of params | compress options |
+| compress.type         | string         | may be `gzip` or `none` |
+| store                 | list of params | storage parameters |
+| store.type            | string         | storage specification; supported types include: `s3`, `stdout` |
+| s3                    | list of params | s3 storage parameters |
+| endpoint              | URL            | address of object store |
+| region                | string         | used to access aws `region` |
+| bucket                | string         | name of bucket in object store where data will be stored |
+| access_key            | string         | credentials for aws-type object store |
+| secret_key            | string         | credentials for aws-type object store |
+| api_key               | string         | credentials for IBM COS-type object store |
```

Comment:
same

Reply:
This is described in the text after the table where there is more elaboration on the keys.

Reply:
maybe embed the below text into the table ... it may be combustion to have both the table and the longer explanation below